internal/driver/glfw/window.go

```diff
@@ -857,7 +873,26 @@ func (w *window) keyPressed(viewport *glfw.Window, key glfw.Key, scancode int, a
 	keyEvent := &fyne.KeyEvent{Name: keyName}
 	keyDesktopModifier := desktopModifier(mods)
 
-	if keyName == fyne.KeyTab {
+	// Activate the menu when the Alt key is pressed
+	focused := w.canvas.Focused()
+	isMenu := false
```

Comment:
This variable is reduntant - inside the ok the menu.active flag could apply to both blocks - the return in the second block means the later check on 895 is not needed

Reply:
Correct - but the simplification was not easy - at least not for me. I an realy impressed that you spotted this so easily.